app/grandchallenge/core/templatetags/bleach.py

```diff
@@ -6,18 +8,45 @@
 from markdown import markdown as render_markdown
 from markdown.extensions.toc import TocExtension
 
-from grandchallenge.core.utils.markdown import LinkBlankTargetExtension
+from grandchallenge.core.utils.markdown import (
+    EmbedYoutubeExtension,
+    LinkBlankTargetExtension,
+)
 
 register = template.Library()
 
 
+def allowed_iframe(tag, name, value):
```

Comment:
Interesting construction. Is this a documented feature of bleach? This library is deprecated, so we want to move away from it. I don't want to add more dependency on features of it as it would make that migration harder. Is it possible to achieve the same with `nh3` https://nh3.readthedocs.io/en/latest/ ?

Reply:
It is a [documented ](https://bleach.readthedocs.io/en/latest/clean.html#using-functions)feature of bleach. I didn't know it was deprecated.

Looking at nh3. It should be nigh equivalent to using `attribute_filter` callables. I don't think it can easily integrate with bleach at the same time, might as well replace it completely then. Shall I bloat the PR with that or run the current bleach implementation and replace it in a new PR?

Reply:
Just investigate, install nh3 in an empty python project and see if it is possible to do the same, and what the solution would look like.

Reply:
Not relevant any more. However, just to comment since I just had my hands on bleach and its settings. I think `nh3` is nigh perfect drop-in replacement for `bleach`.